### PR TITLE
Created line-width class

### PR DESCRIPTION
**Pull request by adamgparsons**
I'm unsure if I have put this in the right place "utilities". Also do I need to compile the css before pushing the branch?

### CODE REVIEW

File: scss/utilities/_u--line-limit.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// ========================================================
+//
+
+// component config
+//
+$line-limit-width: $spacing--md * 37;
+
+// component block
+//
+.line-limit p,
```
Comment (ChrisBrettCA) [inline scss/utilities/_u--line-limit.scss]: `.u-line-limit`

Comment (adamgparsons) [reply scss/utilities/_u--line-limit.scss]: done and added !important

File: scss/utilities/_u--line-limit.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// ========================================================
+// @utility: line-limit
+// ========================================================
+//
+
+// component config
+//
+$line-limit-width: $spacing--md * 37;
```
Comment (ChrisBrettCA) [inline scss/utilities/_u--line-limit.scss]: @adamgparsons where did this number come from, could we comment this line so that we know for future ref?
Ta.

**Comment by ChrisBrettCA** [comment]
This PR solves issue #58

File: scss/utilities/_u--line-limit.scss (hunk 2 above)
Comment (ChrisBrettCA) [reply scss/utilities/_u--line-limit.scss]: Nice, that makes perfect sense, 👍. 
This resolves #51

**Pull request merged**